README copy.md

```diff
@@ -0,0 +1,98 @@
+# Recalls API tester
+
+A starting pattern for AWS lambda in Typescript 
+
+## Requirements
+
+- node v16.17.1
+- [SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html)
+- npm 8+
+
+### Prerequisites
+- Create a `.env`
+    ```shell
+    cp .env.example .env
+    ```
+
+## Build
+
+- `npm i`
+- `npm run build:dev`
+
+### Run using sam local
+Ensure build steps have been completed.
+
+1. `npm run start:dev`
+1. The lambdas should be available at `http://localhost:3000/`
+
+###  To run recalls API tester locally
+To run the recalls faker api locally, you will need to use Postman to send requests at `http://localhost:3000/`. A [postman collection](Postman/Recalls.postman_collection.json) has been included with this repo.
```

Comment:
should this say mock api rather than faker?

Reply:
changed